server/src/main/java/io/spine/server/aggregate/Aggregate.java

```diff
@@ -406,9 +418,8 @@ protected int uncommittedEventsCount() {
      *  @param events the list of the events
      */
     private void addToHistory(Collection<Event> events) {
-        final List<AggregateEventRecord> records = toAggregateEventRecords(events);
-        for (AggregateEventRecord record : records) {
-            historySinceLastSnapshot.addFirst(record);
+        for (Event event : events) {
+            historySinceLastSnapshot.addFirst(event);
```

Comment:
Can we have `recentHistory` instead of `historySinceLastSnapshot`? This way we would be able to extend the notion of the recency without renaming. For example, for some aggregates we may want to have the history until the grandfather of the last snapshot.